parser/types/etc.go

```diff
@@ -121,6 +121,8 @@ func TypeToStr(tp byte, cs string) (r string) {
 		ts = strings.Replace(ts, "text", "blob", 1)
 	} else if IsTypeChar(tp) {
 		ts = strings.Replace(ts, "char", "binary", 1)
+	} else if tp == mysql.TypeNull {
+		ts = "binary"
```

Comment:
So every `null` becomes `binary`? Why not change `type2Str`?

Reply:
You are right. Done.

Reply:
I revert this. I find if null as binary, ```StrToType``` will be output ```"char"```. 

I think it is a problem with the information in infoschema. so that giving the right output can fix this problem.